crates/rethnet_evm/benches/state.rs

```diff
@@ -11,7 +11,7 @@ use util::{bench_sync_state_method, prep_no_op, STORAGE_SCALES};
 fn bench_basic(c: &mut Criterion) {
     bench_sync_state_method(
         c,
-        "StateRef::basic()",
```

Comment:
I think we probably should keep the trait names, because it seems that the "group name" is combined with the "function name" when the benchmark is titled on the CLI and in the HTML report. This line you have highlighted here is used as the group name. Here's an example of a full title as it's printed in the CLI output: `Benchmarking StateDebug:storage_root exist acct/Layered,20 accts per chkpt,10 slots per acct,0 snapshots/2000`